### ISSUE TITLE
jvp-related top 125 operators tracking issue

### ISSUE THREAD

**Issue by zou3519** [pytorch/functorch#658]
## jvp x vjp
The following should pass (and not skip!) test_jvpvjp
- [ ] nn.functional.binary_cross_entropy
- [ ] nn.functional.binary_cross_entropy_with_logits
- [ ] max
- [ ] min
- [ ] norm
- [ ] nn.functional.conv2d
- [ ] nn.functional.batch_norm
- [ ] nn.functional.dropout
- [ ] nn.functional.cross_entropy
- [ ] nn.functional.embedding
- [ ] nn.functional.mse_loss
- [ ] nn.functional.softmax
- [ ] nn.functional.leaky_relu
- [ ] nn.functional.layer_norm
- [ ] nn.functional.pad
- [ ] nn.functional.log_softmax
- [ ] nn.functional.dropout2d
- [ ] nn.functional.nll_loss
- [ ] dist

## vmap x jvp
The following should pass (and not skip!) test_vmapjvpall and test_vmapjvpall_has_batch_rule

- [ ] nn.functional.binary_cross_entropy_with_logits
- [ ] nn.functional.binary_cross_entropy
- [ ] nn.functional.dropout2d"),
- [ ] max
- [ ] min
- [ ] dist
- [ ] nn.functional.linear
- [ ] nn.functional.batch_norm

**Comment by samdow** [comment]
Note (because there's no related PR): checked off `binary_cross_entropy` for `vmap` x `vjp`. I think a forward AD rule was added and locally it passes (doesn't skip) both `test_vmapjvpall` and `test_vmapjvpall_has_batch_rule`
